sherlock/internal/api/login/handler_test.go

```diff
@@ -0,0 +1,37 @@
+package login
```

Comment:
What is this testing?

Reply:
handler_test is sorta borrowing the convention from the other (more multi-endpoint) api packages -- this hosts the suite and the other test files (login_test.go here) use it.